src/view/Office.vue

```diff
@@ -456,6 +476,10 @@ export default {
 			this.close()
 		},
 
+		toggleEdit() {
+			this.hasToggledInteractiveEmbedding = true
+		},
+
```

Comment:
Minor naming nitpicks: 
* This does not really toggle. It only sets to true.
* the names do not match well - `Edit` vs. `InteractiveEmbedding`.

How about this?
```suggestion
		enableEditing() {
			this.hasEditingEnabled = true
		},

```

Reply:
Good call, used hasWidgetEditingEnabled now to be more explicit about where its used.